### ISSUE TITLE
Double counting of concrete in 'Final consumption of non-metallic minerals (aggregate materials group)'

### ISSUE THREAD

**Issue by chauenstein** [IndEcol/RECC-ODYM#61]
@stefanpauliuk 
Results reported for 'Final consumption of non-metallic minerals (aggregate materials group)' currently contain both, cement and concr. aggregates, as well as concrete. As cement and concr. aggregates in buildings are derived by splitting the value for concrete in building archetypes, this causes double counting in this case (raw material input smaller than material consumption for non-metallic minerals [and no major secondary material input])

**Comment by stefanpauliuk** [comment]
This is correct @chauenstein! Input data come for concrete, which is split into cement and aggregates, while concrete is kept as well. Hence, F_6_7 (final consumption, which is reported here) cannot be summed up without double-counting. So far, we have not bothered to sum it up, need to discuss this.
Note that only cement and aggregates have a production impact, not concrete, so for all excensions, the impacts of concrete are only accounted for once, via cement and aggregates.
